packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-utils.spec.ts

```diff
@@ -10,3 +10,13 @@ describe('getButtonAttributes()', () => {
     expect(getButtonAttributes(...args)).toMatchSnapshot();
   });
 });
+
+describe('getIconColor()', () => {
+  it.each<Parameters<typeof getIconColor>>([[false], [true]])(
+    'should return correct icon color for isDisabled: %s',
+    (...args) => {
+      const expectedResult = args[0] ? 'contrast-medium' : 'primary';
```

Comment:
Please have a look at other examples of `it.each` where expected results are defined right next to the parameters producing them.

Reply:
Fixed it